src/rates.ts

```diff
@@ -208,9 +209,15 @@ export const getExchangeRates = async (
         delete doc.updated
         const cleanDoc = asCouchDoc(asObject(asString))(doc)
         for (const pair of Object.keys(cleanDoc.doc)) {
+          // Only save USD rates that do not inlcude zeroRates and
+          // falbackConstantRates to redis and couchdb.
           if (
-            currencyCodeArray(pair).includes('iso:USD') ||
-            preferredCryptoFiatPairs.includes(pair)
+            (currencyCodeArray(pair).includes('iso:USD') ||
+              preferredCryptoFiatPairs.includes(pair)) &&
+            !(
+              pair in currencyCodeMaps.zeroRates ||
```

Comment:
This won't work. 'pair' is a string that includes the currency code and the fiat code like "BTC_iso:USD". The zeroRates map keys don't includes fiat codes at all (because all requests, regardless of fiat, will be returned as 0) so this check will never catch one.